### ISSUE TITLE
[ts][pixi] Upgrade to v8 got Uncaught RangeError: Invalid typed array length: 4

### ISSUE THREAD

**Issue by hotyes** [EsotericSoftware/spine-runtimes#2677]
Hi guys, first of all, thanks for the great work.

I tried to upgrade spine pixi to v8 but got the following errors for some of my animation, they are working fine on spine pixi v7 and pixijs v7

```log
Uncaught RangeError: Invalid typed array length: 4
    at new Float64Array (<anonymous>)
    at fastCopy (chunk-5OHMW4FS.js?v=9c3f205a:5902:34)
    at _Spine.transformAttachments (@esotericsoftware_spine-pixi-v8.js?v=9c3f205a:12564:11)
    at _Spine._applyState (@esotericsoftware_spine-pixi-v8.js?v=9c3f205a:12479:10)
    at SpinePipe.addRenderable (@esotericsoftware_spine-pixi-v8.js?v=9c3f205a:12235:11)
    at collectAllRenderablesSimple (chunk-MUAAMEAI.js?v=9c3f205a:740:33)
    at collectAllRenderables (chunk-MUAAMEAI.js?v=9c3f205a:729:5)
    at collectAllRenderablesSimple (chunk-MUAAMEAI.js?v=9c3f205a:748:7)
    at collectAllRenderables (chunk-MUAAMEAI.js?v=9c3f205a:729:5)
    at collectAllRenderablesSimple (chunk-MUAAMEAI.js?v=9c3f205a:748:7)
```

Version:

Spine Editor version: `4.2.29`
@esotericsoftware/spine-pixi-v8: "^4.2.63"
pixi.js: "^8.5.2"

**Comment by davidetan** [comment]
Hello! We're happy we finally released it.

I'm sorry you encountered an issue. While it's quite clear the line of code where the issue occurs, it's quite surprising to me to see that kind of error.
Unfortunately, I'm not able to reproduce it. Is it possible for you to send us the assets? You can upload them here or send an email to contact@esotericsoftware.com.

**Comment by hotyes** [comment]
hi @davidetan email sent, pls check, thank you!

**Comment by davidetan** [comment]
> hi @davidetan email sent, pls check, thank you!

Thanks for sending us the assets. I'll look into it tomorrow 👍

**Comment by hotyes** [comment]
hello @davidetan, do you think it's possible to fix it today or very soon? or any workaround so we can continue the migration? thank you

**Comment by davidetan** [comment]
> hello @davidetan, do you think it's possible to fix it today or very soon? or any workaround so we can continue the migration? thank you

It depends on the issue itself. I'll work on this today 👍

**Comment by davidetan** [comment]
I've pushed a fix to this issue. Thanks for reporting this.
It will be released soon in 4.2.64.

----

Released 🥳

**Comment by hotyes** [comment]
Confirmed it is fixed, thank you !
